app/views/_nav_bar.html.erb

```diff
@@ -15,42 +54,41 @@
       <li class="nav-item">
         <%= link_to "Browse Merchants", merchants_path, :class => "nav-link"  %>
       </li>
+      <li class="nav-item">
+        <%= link_to 'Users', admin_users_path, :class => "nav-link" %>
+      </li>
+      <li class="nav-item">
+        <%= link_to 'Orders', profile_orders_path, :class => "nav-link" %>
+      </li>
+      <li class="nav-item">
+        <%= link_to 'Dashboard', dashboard_path, :class => "nav-link" %>
+      </li>
 
-      <% if current_user && current_user.role == 'merchant' %>
-        <li class="nav-item">
-          <%= link_to 'Dashboard', dashboard_path, :class => "nav-link" %>
-        </li>
-      <% end %>
+    <% else %>
+      <li class="nav-item active">
+        <%= link_to "Home", root_path, :class => "nav-link"%>
+      </li>
+      <li class="nav-item">
+        <%= link_to "Browse Items", items_path, :class => "nav-link" %>
+      </li>
+      <li class="nav-item">
+        <%= link_to "Browse Merchants", merchants_path, :class => "nav-link"  %>
+      </li>
+    <% end %>
 
-      <% if current_user && current_user.role == "admin"  %>
-        <li class="nav-item">
-          <%= link_to 'Users', admin_users_path, :class => "nav-link" %>
-        </li>
-      <% elsif current_user && current_user.role != ("merchant" || "admin") %>
-        <li class="nav-item">
-          <%= link_to "Shopping Cart (#{ @cart.items.count })", cart_path, :class => "nav-link"  %>
-        </li>
-      <% end %>
-      
-      <% if current_user && current_user.role == "default" || 'admin'  %>
-        <li class="nav-item">
-          <%= link_to 'Profile', profile_path, :class => "nav-link"  %>
-        </li>
-        <li class="nav-item">
-          <%= link_to 'Orders', profile_orders_path, :class => "nav-link" %>
-        </li>
-      <% end %>
+    <% if current_user %>
+      <li class="nav-item">
+        <%= link_to "Logout", logout_path, :class => "nav-link", method: :delete %>
+      </li>
+    <% else %>
+      <li class="nav-item">
+        <%= link_to "Login", login_path, :class => "nav-link" %> </li>
+      <li class="nav-item">
+        <%= link_to "Register", register_path, :class => "nav-link"  %> </li>
+      <li class="nav-item">
+        <%= link_to "Shopping Cart (#{ @cart.items.count })", cart_path, :class => "nav-link"  %></li>
+    <% end %>
 
-      <% if current_user %>
-        <li class="nav-item">
-          <%= link_to "Logout", logout_path, :class => "nav-link", method: :delete %>
-        </li>
-      <% else %>
-        <li class="nav-item">
-          <%= link_to "Login", login_path, :class => "nav-link" %> </li>
-        <li class="nav-item">
-          <%= link_to "Register", register_path, :class => "nav-link"  %> </li>
-      <% end %>
     </ul>
```

Comment:
Much clearer. Appreciate you doing this.

Reply:
+1